src/main/resources/logback.xml

```diff
@@ -1,5 +1,5 @@
 
-<configuration debug="true" scan="true">
+<configuration debug="false">
```

Comment:
I suggest leaving scan=true because otherwise we just have no way to switch on debug level in the runtime

Reply:
done